### PR TITLE
Move Apps to ui/components

### PR DESCRIPTION
**Pull request by sean-brydon**
Part of cleaning up V2 folder.

Moves apps from v2 into ui/components folder.

More stories will be added when we move to SB V7 - allows us to use next link easily supporting AppCard that uses `useRouter` currently its a bit of a pain 

https://github.com/calcom/cal.com/issues/6244 was created on the back of this PR. While moving files about i realised a lot of the skeleton loaders acrossed the app are using the skeleton loader dedicated for app store :thisisfine:

### CODE REVIEW

File: apps/web/components/apps/CalendarListContainer.tsx [hunk 1]
```diff
@@ -15,7 +15,7 @@ import {
   List,
   ShellSubHeading,
   showToast,
-  SkeletonLoader,
+  AppSkeletonLoader as SkeletonLoader,
```
Comment (sean-brydon) [inline apps/web/components/apps/CalendarListContainer.tsx:18]: See comment in PR description

File: packages/ui/components/apps/_storybookData.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
+import { AppFrontendPayload as App } from "@calcom/types/App";
+
+export const _SBApps: App[] = [
```
Comment (sean-brydon) [inline packages/ui/components/apps/_storybookData.ts:3]: Mock data - can be ignored and is only used in stories

File: packages/ui/components/form/select/select.stories.mdx [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import { Canvas, Meta, Story, ArgsTable } from '@storybook/addon-docs';
 import { Examples, Example, Note, Title,CustomArgsTable, VariantRow,VariantsTable} from '@calcom/storybook/components'
-import { Icon } from "@calcom/ui/Icon";
+import { Icon } from "@calcom/ui";
```
Comment (sean-brydon) [inline packages/ui/components/form/select/select.stories.mdx:3]: Icon is breaking stories when directly imported - speaking to @JeroenReumkens this has been fixed in a PR he is working on

**Review by PeerRich — COMMENTED** [review]
looks good, tho merge conflicts

**Review by JeroenReumkens — APPROVED** [review]
@sean-brydon  Looks good, one small question remaining.

File: packages/features/ee/sso/page/teams-sso-view.tsx [hunk 4]
```diff
@@ -2,7 +2,8 @@ import { useRouter } from "next/router";
 
 import { HOSTED_CAL_FEATURES } from "@calcom/lib/constants";
 import { trpc } from "@calcom/trpc/react";
-import { getSettingsLayout as getLayout, SkeletonLoader } from "@calcom/ui";
+// TODO(Sean): Figure out why this is using AppSkeletonLoader
```
Comment (sean-brydon) [inline packages/features/ee/sso/page/teams-sso-view.tsx:5]:
```suggestion
```

**Pull request merged**